### ISSUE TITLE
train_census_ctr_model.py运行疑问

### ISSUE THREAD

**Issue by yimaster-tech** [GoldAndRabbit/gold-deep-rank#1]
按照项目运行train_census_ctr_model.py
INFO:tensorflow:Done calling model_fn.
    INFO:tensorflow:Create CheckpointSaverHook.
    INFO:tensorflow:Graph was finalized.
    INFO:tensorflow:Running local_init_op.
    INFO:tensorflow:Done running local_init_op.
    INFO:tensorflow:Saving checkpoints for 0 into ./model_dir/ckpt_dir/model.ckpt.
    INFO:tensorflow:loss = 0.9890392, step = 1
    INFO:tensorflow:global_step/sec: 153.472
    INFO:tensorflow:loss = 1.848564e-09, step = 501 (3.258 sec)
    INFO:tensorflow:global_step/sec: 263.287
    INFO:tensorflow:loss = 2.7345473e-10, step = 1001 (1.899 sec)
    INFO:tensorflow:global_step/sec: 265.398
    INFO:tensorflow:loss = 3.7554106e-11, step = 1501 (1.884 sec)
    INFO:tensorflow:Saving checkpoints for 2000 into ./model_dir/ckpt_dir/model.ckpt.
    INFO:tensorflow:global_step/sec: 252.654
    INFO:tensorflow:loss = 1.8669908e-11, step = 2001 (1.979 sec)
    INFO:tensorflow:Saving checkpoints for 2036 into ./model_dir/ckpt_dir/model.ckpt.
    INFO:tensorflow:Loss for final step: 5.3157163e-11.
    INFO:tensorflow:Calling model_fn.    
    INFO:tensorflow:Saving dict for global step 2036: accuracy = 1.0, auc = 1.0, global_step = 2036, loss = 4.4924506e-10
loss 和accuracy、auc等指标都不太对，请问是为什么呢？
